From: Michael Olbrich <m.olbrich@pengutronix.de>
Date: Mon, 7 Mar 2022 08:35:41 +0100
Subject: [PATCH] fix building with binutils 2.36

'-T' must be used here as well. Otherwise building fails with:

.../bin/ld: warning: guids.lds contains output sections; did you forget -T?

Signed-off-by: Michael Olbrich <m.olbrich@pengutronix.de>
---
 src/include/workarounds.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/include/workarounds.mk b/src/include/workarounds.mk
index 311883427118..a0521686b2bc 100644
--- a/src/include/workarounds.mk
+++ b/src/include/workarounds.mk
@@ -9,7 +9,7 @@ LD_DASH_T := $(shell \
 	if [ "x${LD_FLAVOR}" = xLLD ] ; then \
 		echo '-T' ; \
 	elif [ "x${LD_FLAVOR}" = xGNU ] ; then \
-		if echo "${LD_VERSION}" | grep -q -E '^2\.3[789]|^2\.[456789]|^[3456789]|^[[:digit:]][[:digit:]]' ; then \
+		if echo "${LD_VERSION}" | grep -q -E '^2\.3[6789]|^2\.[456789]|^[3456789]|^[[:digit:]][[:digit:]]' ; then \
 			echo '-T' ; \
 		else \
 			echo "" ; \
